virtualenv_embedded/activate.sh

```diff
@@ -1,5 +1,6 @@
 # This file must be used with "source bin/activate" *from bash*
 # you cannot run it directly
+set -x
```

Comment:
Not sure if people would appreciate suddenly getting debug output from activate

If we go with it we should restore the previous setting on exit at least